apps/mobile-alphabet/ios/MobileAlphabetTests/MobileAlphabetTests.m

```diff
@@ -0,0 +1,66 @@
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+
+#import <React/RCTLog.h>
+#import <React/RCTRootView.h>
+
+#define TIMEOUT_SECONDS 600
+#define TEXT_TO_LOOK_FOR @"Welcome to React"
+
+@interface MobileAlphabetTests : XCTestCase
+
+@end
+
+@implementation MobileAlphabetTests
+
+- (BOOL)findSubviewInView:(UIView *)view matching:(BOOL (^)(UIView *view))test
+{
+  if (test(view)) {
+    return YES;
+  }
+  for (UIView *subview in [view subviews]) {
+    if ([self findSubviewInView:subview matching:test]) {
+      return YES;
+    }
+  }
+  return NO;
+}
+
+- (void)testRendersWelcomeScreen
```

Comment:
Will you actually write tests at this level? If not, can you remove this boilerplate? Or is it leveraged be Detox? It seems odd that it looks for a welcome screen unless it's boilerplate.